CMakeLists.txt

```diff
@@ -130,7 +130,7 @@ endif()
 # Global language settings
 
 if (NOT CMAKE_CXX_STANDARD)
-    set(CMAKE_CXX_STANDARD 11)
+    set(CMAKE_CXX_STANDARD 14)
```

Comment:
I merged this already, so hopefully a rebase makes this part of the diff go away?